index.html

```diff
@@ -1,181 +1,209 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles.css" />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap"
-      rel="stylesheet"
-    />
-    <title>Chiedozie's Portfolio Website</title>
-  </head>
-  <body>
-    <header>
-      <nav class="fluidsContainer">
-        <div class="navContainer">
-          <div class="navbar">
-            <div class="logo">
-              <a href="#" class="my-logo">James Chiedozie James</a>
-            </div>
-            <div>
-              <i class="fas fa-bars"></i>
-            </div>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+  <link rel="stylesheet" href="styles.css" >
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" >
+  <title>Chiedozie's Portfolio Website</title>
+</head>
+
+<body>
+  <header>
+    <nav class="fluidsContainer">
+      <div class="navContainer">
+        <div class="navbar">
```

Comment:
- [ ] It's important to ensure that your website is responsive and displays correctly across different screen sizes. Currently, your website is not responsive according to the project requirements. Please make the necessary updates to your code to ensure responsiveness starting from a minimum mobile screen size of 375px and adapting to a fully responsive desktop version at 768px. This will improve the user experience and accessibility of your website. Keep up the good work! 🌟👏

[Chiedozie-s-Portfolio-Website.webm](https://github.com/chiedozie013/portfolio-microverse/assets/55840999/1372878d-927b-4cf3-b240-047b30686a8e)